src/ui/reducers/timeline.test.ts

```diff
@@ -236,21 +237,25 @@ describe("Redux timeline state", () => {
     });
 
     describe("set end time", () => {
-      it("should focus from the beginning of the recording to the specified end time if no focus region has been set", async () => {
-        await dispatch(actions.setFocusWindowEnd({ time: 65, sync: false }));
-        expect(getFocusWindow(store.getState())).toMatchInlineSnapshot(`
-          Object {
-            "begin": Object {
-              "point": "0",
-              "time": 0,
-            },
-            "end": Object {
-              "point": "65000",
-              "time": 65,
-            },
-          }
-        `);
-      });
+      // TODO for some reason the test setup doesn't seem to work: it should create a new mock replayClient
+      // with its focusWindow set to null for every test but for some reason this test gets a mock replayClient
+      // with a focusWindow left over from another test: the test works when run separately but not when run
+      // together with the other tests in this file.
```

Comment:
Try adding and then re-enabling this test?
```js
  afterEach(() => {
    jest.resetModules();
  });
```

Reply:
That didn't help.

Reply:
Damn. File a follow up Linear issue?

Reply:
FE-1855